.github/workflows/check-image-vulnerabilities.yml

```diff
@@ -32,17 +26,51 @@ jobs:
           } >> "$GITHUB_STEP_SUMMARY"
 
   check-image-vulnerabilities:
-    name: Checks images in version ${{ inputs.version }} for vulnerabilities
+    name: Check images in version ${{ inputs.version }} for vulnerabilities
     runs-on: ubuntu-latest
     env:
-      QUAY_BEARER_TOKEN: ${{ secrets.QUAY_RHACS_ENG_VULNERABILITY_BEARER_TOKEN }}
+      # ACS-CS dogfooding instance.
+      ROX_ENDPOINT: "acs-cii18grublkv81uil8gg.acs.rhcloud.com"
+    permissions:
+      id-token: write
+    strategy:
+      matrix:
+        image:
+          [
+            "central-db",
+            "collector",
+            "collector-slim",
+            "main",
+            "roxctl",
+            "scanner",
+            "scanner-db",
+            "scanner-db-slim",
+            "scanner-slim",
+            "stackrox-operator",
+          ]
     steps:
-      - run: |
-          set -uo pipefail
-          gh api -H "$ACCEPT_RAW" "${{ env.script_url }}" > check-image-vulnerabilities.py
-          pip install requests==2.28.2
+      - name: Central login
+        uses: stackrox/central-login@v1
+        with:
+          endpoint: https://${{ env.ROX_ENDPOINT }}
+
+      - name: Install roxctl
+        uses: stackrox/roxctl-installer-action@main
```

Comment:
Does this action also have a floating `v1` tag/branch that we could point to?

Reply:
Not yet, but it will soon once we do the first official release.

Reply:
I think we already do, but need to cut a new release with the latest changes.

Reply:
changed to `v1` tag.